src/gitguardian-interface/gitguardian-status-bar.ts

```diff
@@ -81,6 +82,11 @@ function getStatusBarConfig(status: StatusBarStatus): StatusBarConfig {
         text: "GitGuardian - Ignored file",
         color: "statusBarItem.warningBackground",
       };
+    case StatusBarStatus.authFailed:
+      return {
+        text: "GitGuardian - Authentication failed",
+        color: "statusBarItem.errorBackground",
```

Comment:
We could add the command that open the SideBar here, because it's were the "useful" informations will be (i.e api and url sources)